sycl/test-e2e/Matrix/joint_matrix_all_sizes_impl.hpp

```diff
@@ -183,23 +179,23 @@ int init_and_multiply() {
 
 int main() {
   int errors = 0;
-  errors += init_and_multiply<bfloat16, float, 2, 1, SG_SZ, 16>();
-  errors += init_and_multiply<bfloat16, float, 2, 2, SG_SZ, 16>();
-  errors += init_and_multiply<bfloat16, float, 2, 3, SG_SZ, 16>();
-  errors += init_and_multiply<bfloat16, float, 2, 4, SG_SZ, 16>();
-  errors += init_and_multiply<bfloat16, float, 2, 5, SG_SZ, 16>();
-  errors += init_and_multiply<bfloat16, float, 2, 6, SG_SZ, 16>();
-  errors += init_and_multiply<bfloat16, float, 2, 7, SG_SZ, 16>();
-  errors += init_and_multiply<bfloat16, float, 2, 8, SG_SZ, 16>();
-
-  errors += init_and_multiply<int8_t, int32_t, 4, 1, SG_SZ, 32>();
-  errors += init_and_multiply<int8_t, int32_t, 4, 2, SG_SZ, 32>();
-  errors += init_and_multiply<int8_t, int32_t, 4, 3, SG_SZ, 32>();
-  errors += init_and_multiply<int8_t, int32_t, 4, 4, SG_SZ, 32>();
-  errors += init_and_multiply<int8_t, int32_t, 4, 5, SG_SZ, 32>();
-  errors += init_and_multiply<int8_t, int32_t, 4, 6, SG_SZ, 32>();
-  errors += init_and_multiply<int8_t, int32_t, 4, 7, SG_SZ, 32>();
-  errors += init_and_multiply<int8_t, int32_t, 4, 8, SG_SZ, 32>();
+  errors += init_and_multiply<bfloat16, float, 2, 1, SN, 16>();
```

Comment:
Minor, call it TN to make like all the tests

Reply:
Unlike other tests, where TN is just global variable or macro, TN in this test is also used as parameter name in function, so I want to distinguish between parameter name and global variable. Using SN as a global looked like less change to me, than changing TN to something else inside test body.